documentation/memory/step3/using_gc_traces.md

```diff
@@ -1,86 +1,229 @@
 # Tracing garbage collection
 
-There's a lot to learn about how the garbage collector works, but if you learn one thing it's that when GC is running, your code is not.
+This guide will go through the fundamentals of garbage collection traces.
 
-You may want to know how often and how long the garbage collection is running.
+By the end of this guide, you'll be able to:
+* Enable traces in your Node.js application
+* Interpret traces
+* Find memory leak source
 
-## Runnig with garbage collection traces
-You can see traces for garbage collection in console output of your process using the `--trace_gc` flag.
+There's probably a lot of stuff to learn about how the garbage collector works, but if you have to know one thing, it's that when GC is running, your code is not.
 
+You may want to know how often and long the garbage collection runs.
+
+## Setup
+
+For the proposal of this guide, we will use a simple web server empowered with [Fastify](https://www.fastify.io/).
```

Comment:
As mentioned before, I strongly recommend using a minimal application. It will decrease the probability of a further change in this document. For instance, `fastify` can land a release that changes the output of this tracing and the user won't be able to reproduce it.

Reply:
>  It will decrease the probability of a further change in this document.

What if we use a native HTTP server?

> For instance, fastify can land a release that changes the output of this tracing...

The leak we tend to catch here is related to a `Set` that continuously grows, so I wonder how a fastify update could impact this? 

Also, If I choose a server it's because it looks like a real-world case, while scripting is often far from that, but maybe it's a bias. Let me know.

Reply:
If you strongly believe the HTTP Server would be better, I will not block it. I'm just raising my point of view as a user as well.